keel-artifact/src/main/kotlin/com/netflix/spinnaker/keel/artifacts/DockerArtifactSupplier.kt

```diff
@@ -125,6 +128,27 @@ class DockerArtifactSupplier(
     return null
   }
 
+  /**
+   * Given a docker artifact and a docker tag, filters out all tags that don't produce exactly one capture
+   * group with the provided regex.
+   *
+   * This means that this will filter out tags like "latest" from the list.
+   */
+  override fun shouldProcessArtifact(artifactDefinition: DeliveryArtifact, artifact: PublishedArtifact): Boolean {
+    return try {
+      if (artifactDefinition is DockerArtifact) {
+        artifact.version != "latest" && TagComparator.parseWithRegex(artifact.version, artifactDefinition.tagVersionStrategy, artifactDefinition.captureGroupRegex) != null
```

Comment:
we don't want to exclude things here based on the comparator logic. All those tags are valid. The only tag that's not valid is "latest"

Reply:
ok, that changes things. Let me fix it.